test/Table_Tests/src/In_Memory/Fan_Out_Spec.enso

```diff
@@ -0,0 +1,29 @@
+from Standard.Base import all
+
+from project.Util import all
+
+import Standard.Table.Internal.Fan_Out
+from Standard.Table import Table
+
+import Standard.Test.Extensions
+from Standard.Test import Test, Test_Suite, Problems
+
+spec =
+    Test.group "Fan_Out" <|
+        Test.specify "can do fan_out_to_columns " <|
```

Comment:
This is already heavily tested in Split_Tokenize_Spec but we might want to bring more tests over from that at some point.